src/main/java/net/sf/jabref/logic/logging/LogMessage.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.logic.logging;
+
+import javafx.beans.Observable;
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+
+import net.sf.jabref.logic.error.LogMessageWithPriority;
+
+/**
+ * This LogMessage will be save all message entries in a ListProperty.
+ * <p></p>
+ * This list will be use late in ErrorConsoleViewModel to filter message after their priority
```

Comment:
Please fix English. Proposal:

This call saves all messages in a list. The list is used in ErrorConsoleViewModel to filter messages according to their priority.